cogs5e/sheetManager.py

```diff
@@ -436,30 +436,77 @@ async def character(self, ctx, *, name: str = None):
 
     @character.command(name="server")
     @commands.guild_only()
-    async def character_server(self, ctx):
+    async def character_server(self, ctx, args=""):
         """
         Sets the current global active character as a server character.
         If the character is already the server character, unsets the server character.
 
+        args:
```

Comment:
Since the function of reset_all is pretty separate from the function of normal `!char server`, I would say it would be better to make it a subcommand, rather than use the args here.

Reply:
Fair enough! Will make documentation much easier to understand. 

Do you think we would reset all and undo the server being set too?

Reply:
It makes the most sense, and there's no reason not to.